infrastructure/nomad/playbooks/deploy.yml

```diff
@@ -495,7 +495,7 @@
             echo "${ALLOC}" | jq --arg address "0x${ADDRESS}" '
```

Comment:
We should change this step to not use a for-loop. An essential property of the mev-commit chain is that only a single account will receive an initial ETH allocation, the contract deployer, which will later fund the allocator contract.